src/components/questionary/questionaryComponents/NumberInput/QuestionaryComponentNumberInput.tsx

```diff
@@ -129,7 +127,6 @@ export function QuestionaryComponentNumber(props: BasicComponentProps) {
         </Grid>
         <Grid item xs={2} className={classes.unitField}>
           <TextField
-            label="Value"
```

Comment:
Not sure why we remove the label property here. It is a bit strange to see input without label.